mart/attack/composer.py

```diff
@@ -7,12 +7,37 @@
 from __future__ import annotations
 
 import abc
+from collections import OrderedDict
 from typing import Any, Iterable
 
 import torch
 
 
-class Composer(abc.ABC):
+class Function(torch.nn.Module):
+    def __init__(self, *args, order=0, **kwargs) -> None:
+        """A stackable function for Composer.
+
+        Args:
+            order (int, optional): The priority number. A smaller number makes a function run earlier than others in a sequence. Defaults to 0.
+        """
+        super().__init__(*args, **kwargs)
+        self.order = order
+
+    @abc.abstractmethod
+    def forward(self, perturbation, input, target) -> None:
```

Comment:
Return type annotation should be `tuple` of whatever types `perturbation, input, target` are.

Reply:
Yes. Added in 7eccbdd7eb4034203997b400bbfc86ba3603321d